README.md

```diff
@@ -20,52 +20,46 @@ Data and results portal for EEA & Norway Grants - data.eeagrants.org
 
 1. Get the docker installation repository and the source code:
 
-        $ git clone git@gitlab.com:eftafmo/eeag.docker.git
-        $ cd eeag.docker
-        $ git clone git@github.com:eftafmo/dataviz.git src
+        $ git clone git@github.com:eftafmo/dataviz.git
+        $ cd dataviz
 
-2. Prepare environment:
+1. Prepare environment:
 
-        $ cd eeag/
-        $ cp docker-compose.override-dev.yml.example docker-compose.override.yml
-        $ cp web/web.dev.env.example web/web.env
-        $ cd ..
+        $ cp docker/docker-compose.override-dev.yml.example docker-compose.override.yml
+        $ cp docker/web.dev.env.example docker/web.env
 
 
-3. Replace database file with the latest version from production
+1. Create local settings:
 
-        $ mkdir db
-        $ scp edwsys@data.eeagrants.org:/var/local/eeag.docker/db/eeag.sqlite3  db/eeag.sqlite3
-
-5. Create local settings:
-
-        $ cd src
         $ cp dv/localsettings.py.example dv/localsettings.py
```

Comment:
the localsettings.py in ./docker is slightly different than the one used for local development. This step seems to be necessary only when running the stack outside docker

Reply:
In local development, the entire source tree is mounted as a volume: https://github.com/eftafmo/dataviz/blob/fa1c4703488b33402665548cc24124c248f91f56/docker/docker-compose.override-dev.yml.example#L11-L12

This means we need to provide a `localsettings.py` file, otherwise the app will fail to start:

```
root@9262c6ed9f5d:/var/local/dataviz# ./manage.py runserver 0.0.0.0:8000
Traceback (most recent call last):
  File "./manage.py", line 22, in <module>
    execute_from_command_line(sys.argv)
  File "/usr/local/lib/python3.6/site-packages/django/core/management/__init__.py", line 364, in execute_from_command_line
    utility.execute()
  File "/usr/local/lib/python3.6/site-packages/django/core/management/__init__.py", line 308, in execute
    settings.INSTALLED_APPS
  File "/usr/local/lib/python3.6/site-packages/django/conf/__init__.py", line 56, in __getattr__
    self._setup(name)
  File "/usr/local/lib/python3.6/site-packages/django/conf/__init__.py", line 41, in _setup
    self._wrapped = Settings(settings_module)
  File "/usr/local/lib/python3.6/site-packages/django/conf/__init__.py", line 110, in __init__
    mod = importlib.import_module(self.SETTINGS_MODULE)
  File "/usr/local/lib/python3.6/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/var/local/dataviz/dv/settings.py", line 199, in <module>
    from .localsettings import *
ModuleNotFoundError: No module named 'dv.localsettings'
```

You could argue that we should copy the docker localsettings file (the same one copied by the Dockerfile), and I'd agree. FWIW, that line was already there in the readme, I followed it when I made my initial setup, and it worked.

Reply:
Should I leave it as-is?

Reply:
Ok